# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/pmac-no-boot-guess.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Disable guessing root from apple partition magic.
We never booted like this ever to start with.

--- linux-6.18/block/partitions/mac.c.vanilla	2025-12-19 21:00:12.863492671 +0100
+++ linux-6.18/block/partitions/mac.c	2025-12-19 21:00:23.104656862 +0100
@@ -11,6 +11,8 @@
 #include "check.h"
 #include "mac.h"
 
+#undef CONFIG_PPC_PMAC
+
 #ifdef CONFIG_PPC_PMAC
 #include <asm/machdep.h>
 extern void note_bootable_part(dev_t dev, int part, int goodness);
